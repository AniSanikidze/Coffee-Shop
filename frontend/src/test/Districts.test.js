// import * as React from "react";
// import * as ReactDOM from "react-dom";
// import * as enzyme from 'enzyme';
// import Adapter from 'enzyme-adapter-react-16'
// import renderer from 'react-test-renderer';
// import { BrowserRouter as Router } from 'react-router-dom'
// import { configure } from 'enzyme';
// import { UserContext } from "../UserContext";

// configure({ adapter: new Adapter() });

// describe('testing CollegeSection', () => {

//     it('should render', () => {
//         const DistrictsComponent = renderer.create(<UserContext.Provider value={false}><Router><Districts /></Router></UserContext.Provider>).toJSON();
//         expect(DistrictsComponent).toMatchSnapshot();
//     })

// })