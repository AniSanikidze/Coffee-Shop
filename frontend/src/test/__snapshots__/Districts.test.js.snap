// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing CollegeSection should render 1`] = `
<div
  className="districts"
>
  <h1>
    Popular districts in 
    <b>
      Prague
    </b>
  </h1>
  <div
    className="districts__container"
  >
    <div
      className="districts__wrapper"
    >
      <ul
        className="districts__items"
      >
        <li
          className="districts__item"
          onClick={[Function]}
        >
          <a
            className="districts__item__link"
            href="/restaurants"
            onClick={[Function]}
          >
            <h4>
              Praha 1
               (
              1062
               places)
            </h4>
            <i
              class="fas fa-angle-right"
            />
          </a>
        </li>
        <li
          className="districts__item"
          onClick={[Function]}
        >
          <a
            className="districts__item__link"
            href="/restaurants"
            onClick={[Function]}
          >
            <h4>
              Praha 2
               (
              387
               places)
            </h4>
            <i
              class="fas fa-angle-right"
            />
          </a>
        </li>
        <li
          className="districts__item"
          onClick={[Function]}
        >
          <a
            className="districts__item__link"
            href="/restaurants"
            onClick={[Function]}
          >
            <h4>
              Praha 4
               (
              296
               places)
            </h4>
            <i
              class="fas fa-angle-right"
            />
          </a>
        </li>
      </ul>
      <ul
        className="districts__items"
      >
        <li
          className="districts__item"
          onClick={[Function]}
        >
          <a
            className="districts__item__link"
            href="/restaurants"
            onClick={[Function]}
          >
            <h4>
              Praha 5
               (
              310
               places)
            </h4>
            <i
              class="fas fa-angle-right"
            />
          </a>
        </li>
        <li
          className="districts__item"
          onClick={[Function]}
        >
          <a
            className="districts__item__link"
            href="/restaurants"
            onClick={[Function]}
          >
            <h4>
              Praha 6
               (
              222
               places)
            </h4>
            <i
              class="fas fa-angle-right"
            />
          </a>
        </li>
        <li
          className="districts__item"
          onClick={[Function]}
        >
          <a
            className="districts__item__link"
            href="/restaurants"
            onClick={[Function]}
          >
            <h4>
              Praha 10
               (
              259
               places)
            </h4>
            <i
              class="fas fa-angle-right"
            />
          </a>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
